### ISSUE TITLE
[Bug]: Opening a modal creates a console error

### ISSUE THREAD

**Issue by ArakTaiRoth** [GovAlta/ui-components#1248]
### Describe the bug

When you open a modal, two errors show up in the console:
`Uncaught TypeError: Cannot read properties of undefined (reading 'addEventListener')`
`Uncaught TypeError: Cannot read properties of undefined (reading 'offsetHeight')`

### Provide Code

Issue can be reproduced in the Storybook

### To Reproduce

Go to this [page](https://ui-components.alberta.ca/?path=/docs/components-modal--basic) and open any modal with the developer console open.

### Relevant console log output

```shell
main.88c36abe.iframe.bundle.js:1 Uncaught TypeError: Cannot read properties of undefined (reading 'addEventListener')
    at main.88c36abe.iframe.bundle.js:1:184877
    at run (main.88c36abe.iframe.bundle.js:1:37826)
    at Array.map (<anonymous>)
    at FocusTrap.connectedCallback (main.88c36abe.iframe.bundle.js:1:53105)
    at insert (main.88c36abe.iframe.bundle.js:1:39827)
    at Object.m (main.88c36abe.iframe.bundle.js:1:275516)
    at Object.p (main.88c36abe.iframe.bundle.js:1:280123)
    at update (main.88c36abe.iframe.bundle.js:1:44307)
    at flush (main.88c36abe.iframe.bundle.js:1:43626)
(anonymous) @ main.88c36abe.iframe.bundle.js:1
run @ main.88c36abe.iframe.bundle.js:1
connectedCallback @ main.88c36abe.iframe.bundle.js:1
insert @ main.88c36abe.iframe.bundle.js:1
m @ main.88c36abe.iframe.bundle.js:1
p @ main.88c36abe.iframe.bundle.js:1
update @ main.88c36abe.iframe.bundle.js:1
flush @ main.88c36abe.iframe.bundle.js:1
Promise.then (async)
schedule_update @ main.88c36abe.iframe.bundle.js:1
make_dirty @ main.88c36abe.iframe.bundle.js:1
(anonymous) @ main.88c36abe.iframe.bundle.js:1
(anonymous) @ main.88c36abe.iframe.bundle.js:1
setTimeout (async)
$$self.$$.update @ main.88c36abe.iframe.bundle.js:1
update @ main.88c36abe.iframe.bundle.js:1
flush @ main.88c36abe.iframe.bundle.js:1
set open @ main.88c36abe.iframe.bundle.js:1
attributeChangedCallback @ main.88c36abe.iframe.bundle.js:1
qa @ 6198.92c98c5f.iframe.bundle.js:2
ij @ 6198.92c98c5f.iframe.bundle.js:2
dk @ 6198.92c98c5f.iframe.bundle.js:2
exports.unstable_runWithPriority @ 6198.92c98c5f.iframe.bundle.js:2
gg @ 6198.92c98c5f.iframe.bundle.js:2
Uj @ 6198.92c98c5f.iframe.bundle.js:2
Lj @ 6198.92c98c5f.iframe.bundle.js:2
(anonymous) @ 6198.92c98c5f.iframe.bundle.js:2
exports.unstable_runWithPriority @ 6198.92c98c5f.iframe.bundle.js:2
gg @ 6198.92c98c5f.iframe.bundle.js:2
jg @ 6198.92c98c5f.iframe.bundle.js:2
ig @ 6198.92c98c5f.iframe.bundle.js:2
Jg @ 6198.92c98c5f.iframe.bundle.js:2
Oh @ 6198.92c98c5f.iframe.bundle.js:2
onClick @ main.88c36abe.iframe.bundle.js:1
listener @ main.88c36abe.iframe.bundle.js:1
clickHandler @ main.88c36abe.iframe.bundle.js:1
main.88c36abe.iframe.bundle.js:1 Uncaught TypeError: Cannot read properties of undefined (reading 'offsetHeight')
    at main.88c36abe.iframe.bundle.js:1:325159
    at run (main.88c36abe.iframe.bundle.js:1:37826)
    at Array.map (<anonymous>)
    at Scrollable.connectedCallback (main.88c36abe.iframe.bundle.js:1:53105)
    at append (main.88c36abe.iframe.bundle.js:1:39362)
    at Object.m (main.88c36abe.iframe.bundle.js:1:275868)
    at Object.p (main.88c36abe.iframe.bundle.js:1:280123)
    at update (main.88c36abe.iframe.bundle.js:1:44307)
    at flush (main.88c36abe.iframe.bundle.js:1:43626)
```

### Additional context

There's also a line in the console for the Storybook that says `GoAModal [type] is deprecated.`. Might want to fix that at the same time.

**Comment by chrisolsen** [comment]
This bug was "accidentally" fixed in https://github.com/GovAlta/ui-components/pull/1263/commits/0aa35ce83f6428c2599c084391734816a00553a1

**Comment by ArakTaiRoth** [comment]
Just to add to  @chrisolsen 's comment, this was fixed in this PR, #1263
